x-pack/plugins/cases/server/routes/api/cases/push_case.ts

```diff
@@ -32,11 +32,6 @@ export function initPushCaseApi({ router, logger }: RouteDeps) {
         }
 
         const casesClient = await context.cases.getCasesClient();
-        const actionsClient = context.actions?.getActionsClient();
```

Comment:
The actions client retrieved using the factory's method so we don't need it here anymore.